### PR TITLE
[F] LRAC-14755 AC users with member permission cannot access the Suppressed Users list

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LRAC-14755

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/pages/SuppressedUsers.tsx [hunk 1]
```diff
@@ -57,4 +58,10 @@ export const SuppressedUsers: React.FC<ISuppressedUsersProps> = ({
...
 };
 
-export default compose<any>(withAdminPermission, connector)(SuppressedUsers);
+const withAllPermissions = withPermission([
+	UserRoleNames.Administrator,
+	UserRoleNames.Member,
+	UserRoleNames.Owner
+]);
+
+export default compose<any>(withAllPermissions, connector)(SuppressedUsers);
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/pages/SuppressedUsers.tsx:67]:
```suggestion

export default SuppressedUsers;
```

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/pages/SuppressedUsers.tsx (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/pages/SuppressedUsers.tsx]: @interaminense done! :-)

**Comment by fmarcio** [comment]
ci:test:sf

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @fmarcio

**Comment by interaminense** [comment]
ci:forward

**Comment by fmarcio** [comment]
ci:test:stable

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

**Comment by ViniLopes20** [comment]
ci:test:stable

**Comment by ViniLopes20** [comment]
ci:test:stable

**Comment by fmarcio** [comment]
ci:test:stable

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**